workflows/prognostic_c48_run/runtime/steppers/interval.py

```diff
@@ -0,0 +1,36 @@
+import cftime
+from datetime import timedelta
+from typing import Tuple
+import xarray as xr
+
+from runtime.types import Diagnostics
+from runtime.steppers.stepper import Stepper
+
+
+class IntervalStepper:
+    def __init__(self, apply_interval_seconds: float, stepper: Stepper):
+        self.start_time = None
+        self.interval = timedelta(seconds=apply_interval_seconds)
+        self.stepper = stepper
+
+    def _need_to_update(self, time: cftime.DatetimeJulian):
+        if self.start_time is not None:
```

Comment:
> The pattern we use for prescribing data that exists only at some interval in disk (maybe analogous to what you're doing with prescribing at intervals here) is to specify its start time and interval from config for clarity. Would that make sense here?

`start_time` in this context is different from the prescriber config's "start_time" since here it's referring to the start time of the simulation, not the reference data. For the case where this is wrapping a `Prescriber`, this information about the dataset bound would already be in the `base_config` part of the `IntervalConfig`. (I think that's what you're referring to in the comment but if not, ignore)

>  I'm not clear on how time == self.start_time could ever be true?

That's correct, removed. It was leftover from some earlier iteration of the code.